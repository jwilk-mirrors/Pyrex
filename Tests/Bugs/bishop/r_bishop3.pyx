#cdef extern from "r_bishop3_debug.h":
#	void bugrit(char *, type)

cdef class Foo:
	cdef int val
	cdef object obj

	def __init__(self):
		self.val = 0


cdef class Fee(Foo):

	def bof(self):
		print 'Fee bof', self.val


cdef class Faa(Fee):

	def bof(self):
		print 'Foo bof', self.val

#bugrit("Foo", Foo)
#bugrit("Fee", Fee)
#bugrit("Faa", Faa)
