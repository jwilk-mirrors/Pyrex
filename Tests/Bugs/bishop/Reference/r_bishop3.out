Fee bof 0
Foo bof 0
